llvm/include/llvm/ADT/EquivalenceClasses.h

```diff
@@ -184,6 +185,14 @@ class EquivalenceClasses {
     return member_iterator(nullptr);
   }
 
+  iterator_range<member_iterator> members(const ECValue &ECV) const {
+    return make_range(member_begin(ECV), member_end());
+  }
+
+  iterator_range<member_iterator> members(const ElemTy &V) const {
```

Comment:
Is that variant used by anything other than the newly added unit test? If not, it might be better to update the unit test to use the ECValue variant and remove the function here?

Reply:
Yeah, it's used in LAA.

Reply:
Ah right I guess if it is just a single use it could just do `members(findLeader(V))`?

Reply:
Yeah, I had that initially, but I think the new API is clean, and makes a lot of intuitive sense.

Reply:
Oh wait, findLeader returns an iterator and not an ECValue, so that's a problem.

Reply:
While I agree with your comment in principle, what I had earlier was members(getLeaderClass(V)), as none of the APIs return an ECValue, but figured that this was more trouble than it was worth.

I agree, EquivClass is already quite inconsistent, and I see that you also have some recent changes improving it: we can probably improve it some more, when we have some free cycles.